frontend/src/App.tsx

```diff
@@ -713,11 +732,18 @@ export class App extends PureComponent<Props, State> {
       RERUN_PROMPT_MODAL_DIALOG &&
       sessionEvent.type === "scriptChangedOnDisk"
     ) {
+      const allowRunOnSave =
```

Comment:
This is a dead code because the variable RERUN_PROMPT_MODAL_DIALOG is always set to False. but I would prefer to deal with this problem in a separate PR, because we have to consider whether we can delete it.